public/router.php

```diff
@@ -25,15 +25,21 @@
 	date_default_timezone_set('UTC');
 }
 
-define('BASE_DIR', dirname(__DIR__).'/');
+$baseDir = str_replace('\\', '/', dirname(dirname(__FILE__)).'/');
+if (!is_file($baseDir.'pufferpanel')) {
+    $baseUrl = dirname($_SERVER['PHP_SELF']);
```

Comment:
And why is knowing how the files are structured required? I need a technical reason for this.